velox/common/file/CMakeLists.txt

```diff
@@ -17,6 +17,10 @@ include_directories(.)
 add_library(velox_file File.cpp FileSystems.cpp Utils.cpp)
 target_link_libraries(velox_file velox_common_base Folly::folly)
 
+if(NOT VELOX_DISABLE_GOOGLETEST)
```

Comment:
Why this change? Why does velox_file depend on gtest?

Reply:
I deleted the `append(dwio::common::DataBuffer<char>& buffer)` api, so this dependency can be deleted.